compiler/src/iree/compiler/Codegen/Common/ConcretizePadResultShape.cpp

```diff
@@ -132,16 +132,18 @@ class ConcretizePadResultShapePass final
     : public impl::ConcretizePadResultShapePassBase<
           ConcretizePadResultShapePass> {
 public:
+  LogicalResult initialize(MLIRContext *context) override {
+    config.listener = &listener;
+    return success();
+  }
+
   void runOnOperation() override {
     MLIRContext *context = &getContext();
     auto funcOp = getOperation();
```

Comment:
Instead, could we pull out this config to the outer block?

Reply:
How's this?